### ISSUE TITLE
Login and registration forms are missing autocomplete (WCAG 2.1 SC 1.3.5, level AA)

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/front-end-monorepo#5646]
## Package
_Choose from the list:_
- [x] app-content-pages
- [x] app-project
- [ ] lib-classifier
- [ ] lib-panoptes-js
- [ ] lib-react-components
- [ ] unknown

## Describe the bug
WCAG 2.1 requires autocomplete on text inputs for user account info, to meet [SC 1.3.5: Identify input purpose](https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html).

## Expected behavior
Text inputs for personal information should have autocomplete eg. `autocomplete='email'`.

**Comment by eatyourgreens** [comment]
Now that #5657 has merged, this is fixed for the login and registration forms on the Team and Publications pages.
